deps/src/main/java/com/microsoft/azure/sdk/iot/deps/transport/amqp/CbsAuthorizationType.java

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.azure.sdk.iot.deps.transport.amqp;
+
+/**
+ * An enumeration of supported authorization methods for AMQP connections that use Claims Based Security links.
+ */
+public enum CbsAuthorizationType
+{
+    /**
+     * Authorize with CBS through a shared access signature.
+     */
+    SHARED_ACCESS_SIGNATURE("azure-devices.net:sastoken");
```

Comment:
Currently the only token auth type Hub supports, but this is where we'd add an enum for RBAC style auth tokens later